.github/ISSUE_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+# Overview
+_What is the purpose of this pull request?_
+
+# Implementation
+_What steps were taken to achieve/implement this pull request?_
+
+_Example:_
+1. Ran `rails g scaffold University` to generate a university controller, model and views. Before running this, I had to run this to disable spring temporarily: `export DISABLE_SPRING=true `
+
+# Testing (Manual)
+_What are the steps for manual verification that the changes in this pull request work as expected?_
+
+_Example:_
+1. Run `rails server`
+2. Access `http://localhost:3000/universities` to view a list of universities
+3. On the same page, you can click "New University" to create a new university, and more.
+
+
+# Testing (Automated)
+_What happy paths have been tested?_
+
+_What sad paths have been tested?_
+
```

Comment:
Can we add a section somewhere to describe any blockers/problems you ran into and the steps/resources used to fix them? 
```suggestion
# Problems Faced
_Did you run into any problems, if so how did you resolve them?
```

Reply:
That is a good idea, lets add that in there.